src/Jss.js

```diff
@@ -15,6 +15,11 @@ export default class Jss {
     this.sheets = new SheetsRegistry()
     this.plugins = new PluginsRegistry()
     this.uid = uid
+    if (__VERSION__) {
```

Comment:
Another option would be to just use a custom script or a lib which just runes over lib and dist files and replaces something. And call it in the "scripts" commands